src/Task/PhpStan.php

```diff
@@ -73,6 +73,14 @@ public function run(ContextInterface $context): TaskResultInterface
             ->notPaths($config['ignore_patterns'])
             ->extensions($config['triggered_by']);
 
+        if (!($config['configuration'])) {
```

Comment:
https://phpstan.org/config-reference#config-file

> If you do not provide a config file explicitly, PHPStan will look for files named phpstan.neon or phpstan.neon.dist in current directory.

Why don't we let the tool decide? It detects these files by default - no need to maintain it in here.
If you want to specify a custom location, you can always do so in your configuration.
If you want to use the defaults, you can just net set the configuration option (which will fallback to the phpstan defaults.)

Reply:
Sorry, I was wrong about that. My mistake. 
After further testing, it does indeed take the phpstan.neon file well even if we don't specify it.

Reply:
No problem. :)